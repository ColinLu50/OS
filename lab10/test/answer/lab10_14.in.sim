6970
423
285
237
273
230
