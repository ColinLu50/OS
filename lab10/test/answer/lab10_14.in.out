FCFS
47 -> 25 -> 151 -> 157 -> 163 -> 137 -> 71 -> 6 -> 54 -> 162 -> 83 -> 33 -> 170 -> 52 -> 122 -> 131 -> 104 -> 90 -> 175 -> 200 -> 20 -> 231 -> 147 -> 17 -> 224 -> 64 -> 238 -> 205 -> 9 -> 208 -> 38 -> 199 -> 141 -> 67 -> 213 -> 49 -> 161 -> 236 -> 163 -> 227 -> 56 -> 96 -> 81 -> 86 -> 51 -> 216 -> 124 -> 134 -> 20 -> 138 -> 138 -> 54 -> 95 -> 128 -> 165 -> 49 -> 70 -> 23 -> 185 -> 98 -> 182 -> 52 -> 63 -> 112 -> 218 -> 216 -> 137 -> 25 -> 129 -> 219 -> 189 -> 116 -> 42 -> 169 -> 123 -> 65 -> 128 -> 212 -> 52 -> 187 -> 73 -> 161
6970
SSTF
47 -> 49 -> 49 -> 51 -> 52 -> 52 -> 52 -> 54 -> 54 -> 56 -> 63 -> 64 -> 65 -> 67 -> 70 -> 71 -> 73 -> 81 -> 83 -> 86 -> 90 -> 95 -> 96 -> 98 -> 104 -> 112 -> 116 -> 122 -> 123 -> 124 -> 128 -> 128 -> 129 -> 131 -> 134 -> 137 -> 137 -> 138 -> 138 -> 141 -> 147 -> 151 -> 157 -> 161 -> 161 -> 162 -> 163 -> 163 -> 165 -> 169 -> 170 -> 175 -> 182 -> 185 -> 187 -> 189 -> 199 -> 200 -> 205 -> 208 -> 212 -> 213 -> 216 -> 216 -> 218 -> 219 -> 224 -> 227 -> 231 -> 236 -> 238 -> 42 -> 38 -> 33 -> 25 -> 25 -> 23 -> 20 -> 20 -> 17 -> 9 -> 6
423
SCAN
47 -> 42 -> 38 -> 33 -> 25 -> 23 -> 20 -> 17 -> 9 -> 6 -> 0 -> 49 -> 51 -> 52 -> 54 -> 56 -> 63 -> 64 -> 65 -> 67 -> 70 -> 71 -> 73 -> 81 -> 83 -> 86 -> 90 -> 95 -> 96 -> 98 -> 104 -> 112 -> 116 -> 122 -> 123 -> 124 -> 128 -> 129 -> 131 -> 134 -> 137 -> 138 -> 141 -> 147 -> 151 -> 157 -> 161 -> 162 -> 163 -> 165 -> 169 -> 170 -> 175 -> 182 -> 185 -> 187 -> 189 -> 199 -> 200 -> 205 -> 208 -> 212 -> 213 -> 216 -> 218 -> 219 -> 224 -> 227 -> 231 -> 236 -> 238
285
C-SCAN
47 -> 42 -> 38 -> 33 -> 25 -> 23 -> 20 -> 17 -> 9 -> 6 -> 0 -> 239 -> 238 -> 236 -> 231 -> 227 -> 224 -> 219 -> 218 -> 216 -> 213 -> 212 -> 208 -> 205 -> 200 -> 199 -> 189 -> 187 -> 185 -> 182 -> 175 -> 170 -> 169 -> 165 -> 163 -> 162 -> 161 -> 157 -> 151 -> 147 -> 141 -> 138 -> 137 -> 134 -> 131 -> 129 -> 128 -> 124 -> 123 -> 122 -> 116 -> 112 -> 104 -> 98 -> 96 -> 95 -> 90 -> 86 -> 83 -> 81 -> 73 -> 71 -> 70 -> 67 -> 65 -> 64 -> 63 -> 56 -> 54 -> 52 -> 51 -> 49
237
LOOK
47 -> 42 -> 38 -> 33 -> 25 -> 23 -> 20 -> 17 -> 9 -> 6 -> 49 -> 51 -> 52 -> 54 -> 56 -> 63 -> 64 -> 65 -> 67 -> 70 -> 71 -> 73 -> 81 -> 83 -> 86 -> 90 -> 95 -> 96 -> 98 -> 104 -> 112 -> 116 -> 122 -> 123 -> 124 -> 128 -> 129 -> 131 -> 134 -> 137 -> 138 -> 141 -> 147 -> 151 -> 157 -> 161 -> 162 -> 163 -> 165 -> 169 -> 170 -> 175 -> 182 -> 185 -> 187 -> 189 -> 199 -> 200 -> 205 -> 208 -> 212 -> 213 -> 216 -> 218 -> 219 -> 224 -> 227 -> 231 -> 236 -> 238
273
C-LOOK
47 -> 42 -> 38 -> 33 -> 25 -> 23 -> 20 -> 17 -> 9 -> 6 -> 238 -> 236 -> 231 -> 227 -> 224 -> 219 -> 218 -> 216 -> 213 -> 212 -> 208 -> 205 -> 200 -> 199 -> 189 -> 187 -> 185 -> 182 -> 175 -> 170 -> 169 -> 165 -> 163 -> 162 -> 161 -> 157 -> 151 -> 147 -> 141 -> 138 -> 137 -> 134 -> 131 -> 129 -> 128 -> 124 -> 123 -> 122 -> 116 -> 112 -> 104 -> 98 -> 96 -> 95 -> 90 -> 86 -> 83 -> 81 -> 73 -> 71 -> 70 -> 67 -> 65 -> 64 -> 63 -> 56 -> 54 -> 52 -> 51 -> 49
230
